src/pages/GenerateKeys/Option2.tsx

```diff
@@ -0,0 +1,283 @@
+        </pre>
+      );
+    }
+  };
+
+  return (
+    <Paper className="mt20">
+      <Heading level={2} size="small" color="blueMedium" className="mb20">
+        Option 2: build deposit-cli from the Python source code
+      </Heading>
+      <Heading level={4} size="small" color="blueMedium" className="mb10">
+        Install python3.7+
+      </Heading>
+      <Text>{renderPythonInstructions()}</Text>
+      <Text>
+        For the most up-to-date installation instructions, please visit{' '}
+        <Link
+          primary
+          inline
+          external
+          to="https://python.org/about/gettingstarted"
+        >
+          https://python.org/about/gettingstarted
+        </Link>
+        .
+      </Text>
+      <Alert variant="info" className="my10">
+        <Text className="my10" color="blueDark">
+          You can check your Python version by typing{' '}
+          <span className="alert-highlight">python3 -v</span> on your terminal.
+        </Text>
+      </Alert>
+
+      <Heading level={4} size="small" color="blueMedium" className="mb10 mt20">
+        Install pip3
+      </Heading>
+      {renderPipInstructions()}
+      <Heading level={4} size="small" color="blueMedium" className="mb10 my20">
+        Install virtualenv
+      </Heading>
+      <Text>
+        virtualenv would help you to create an isolated Python environment for
+        deposit-cli tool. Please visit{' '}
+        <Link
+          external
+          to="https://virtualenv.pypa.io/en/latest/installation.html"
+          inline
+          primary
+        >
+          https://virtualenv.pypa.io/en/latest/installation.html
+        </Link>
+      </Text>
+      <Heading level={4} size="small" color="blueMedium" className="mb10 mt20">
+        Install deposit-cli tool
+      </Heading>
+      <Text>
+        Download and uncompress the{' '}
+        <Link
+          external
+          inline
+          primary
+          to="https://github.com/ethereum/eth2.0-deposit-cli/releases/download/v0.2.1/eth2deposit-cli-v0.2.1-darwin-amd64.tar.gz"
+        >
+          source code{' '}
+        </Link>
+        from GitHub.
+      </Text>
+      <Alert variant="info" className="my10">
+        <Text>
+          If you’re a git user, you can run{' '}
+          <span className="alert-highlight">
+            git clone https://github.com/ethereum/eth2.0-deposit-cli.git
+          </span>
+        </Text>
+      </Alert>
+      <Text>First, create an venv virtualenv:</Text>
+      <Alert variant="secondary" className="my10">
+        <pre className="my0">
+          <span style={{ color: colors.blue.medium }}>virtualenv</span> venv
+        </pre>
+        <pre className="my0">./venv/bin/activate</pre>
+      </Alert>
+      <Text>Second, install the dependency packages:</Text>
+      <Alert variant="secondary" className="my10">
+        <pre className="my0">
+          <span style={{ color: colors.red.medium }}>python3 setup</span>
```

Comment:
A forking here:

For Linux/macOS users, the same:
```
          <span style={{ color: colors.red.medium }}>python3 setup</span>
```

But for Windows users (assume `python` is python 3):

```
          <span style={{ color: colors.red.medium }}>python setup</span>
```